src/env.d.ts

```diff
@@ -34,8 +34,23 @@ declare module NodeJS {
     }
 }
 
+interface Clipboard extends EventTarget {
+    write(data: ClipboardItem[]): Promise<void>
+}
+
+declare class ClipboardItem {
+    constructor(data: { [mimeType: string]: Blob })
+}
+
+type PermissionNameWithClipboard = PermissionName | 'clipboard-read' | 'clipboard-write'
```

Comment:
https://github.com/microsoft/TypeScript/pull/34562

It can be removed after https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/930